Tests/DefaultModelProviderFactoryTests.cs

```diff
@@ -31,49 +36,65 @@ static DefaultModelProviderFactoryTests()
         }
 
 
+
         private readonly ITestOutputHelper testOutputHelper;
 
         public DefaultModelProviderFactoryTests(ITestOutputHelper  testOutputHelper)
         {
             this.testOutputHelper = testOutputHelper;
+            // Create private instance of DI
+            xbimServices = XbimServices.CreateInstance();
         }
 
-        [Fact]
+        XbimServices xbimServices;
+
+        [Fact(Skip = "Need to re-consider the requirement - should it be explicit")]
```

Comment:
Skipped - needs a design decision. When Essent is in the appdomain the Factory would use Heuristic rather than MemoryModelProvider

Now it needs to be specified explicitly. But also since we can now deploy Esent in net core we could re-intro to Xbim.Ifc and make the default again.